nunit-install.sln

```diff
@@ -1,41 +0,0 @@
-﻿
-Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Can't we just stick with `nunit-install.sln` instead of renaming it to runners? We are dropping the framework project from the solution, so there is no need for multiple solutions.